examples/xonotic/Makefile

```
@@ -23,12 +23,12 @@
 #     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
 #
 
-REPOSITORY = us-docker.pkg.dev/agones-images/examples
+REPOSITORY ?=
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
 root_path := $(realpath $(project_path)/../..)
-image_tag = $(REPOSITORY)/xonotic-example:1.1
+image_tag = xonotic-example:1.1
```

Pretty sure this will work:

```suggestion
image_tag = xonotic-example:1.1
ifneq ($(REPOSITORY),)
  image_tag := $(REPOSITORY)/${image_tag)
endif
```

Then you don't have to add it all the way through, and also, a tag can't start with "/".

---

Added an if condition for image_tag